### ISSUE TITLE
Port Edit Profile

### ISSUE THREAD

**Issue by Mihakuu** [HabitRPG/habitica#1459]
I jumped in to change my profile picture but it does not update. Also investigated and nothing can be changed in the profile section.
upon refresh everything is back to what it was before.
Work around is going to old.habitrpg but something will have to fixed later on

**Comment by lefnire** [comment]
TBP (to be ported)

**Comment by lefnire** [comment]
i'll work on this now

**Comment by lefnire** [comment]
pushing to beta. play around with it a bit, then i'll merge to prod

**Comment by paglias** [comment]
Works but I would add a save button because now unless you click "Sync", make another request to the server or add a webdite then changes are not persisted

**Comment by lefnire** [comment]
it's saved on ng-blur. so the only time it won't get saved is if they edit, and then refresh or leave the page without leaving the input field. I figured that case was highly unlikely, but it's still really bad. I could do ng-change, but that will be a big performance issue. Is there some sort of "ng-they've-stopped-typing"? 

Yeah, maybe I should just add a save button after all. It just felt clunky when I was playing around with it (I added that initially)
